app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
+    if (len(command_list) == 3
+            and command_list[1] != command_list[2]
+            and command_list[0] == "cp"):
```

Comment:
couldu unpack ur vars like this
`arg1, arg2, arg3 = command.split()`
to avoid using indexes and increase readability